app/Http/Controllers/TrainingDash.php

```diff
@@ -549,7 +555,7 @@ public function completeOTS(Request $request, $id) {
             $audit = new Audit;
             $audit->cid = Auth::id();
             $audit->ip = $_SERVER['REMOTE_ADDR'];
-            $audit->what = Auth::user()->full_name.' updated an OTS for '.User::find($ots->controller_id)->full_name.'.';
+            $audit->what = Auth::user()->full_name . ' updated an OTS for ' . User::find($ots->controller_id)->full_name . '.';
             $audit->save();
```

Comment:
I think what's happening with this formatting is there are probably a few things it only checks in changed files because checking everywhere would take forever. So the formatting around the concat "." must be one of those. I could be wrong but I've seen this with JS's pretty-quick.

Reply:
Got it - that makes sense. This isn't a change that I made, so it's being done by the formater